drivers/hmis/app/graphql/types/hmis_schema/project_input.rb

```diff
@@ -0,0 +1,28 @@
+module Types
+  class HmisSchema::ProjectInput < BaseInputObject
+    description 'HMIS Project input'
+
+    argument :organization_id, ID, required: false
+    argument :project_name, String, required: false
+    date_string_argument :operating_start_date, 'Date with format yyyy-mm-dd', required: false
+    date_string_argument :operating_end_date, 'Date with format yyyy-mm-dd', required: false
+    argument :description, String, required: false
+    argument :contact_information, String, required: false
+    argument :project_type, Types::HmisSchema::Enums::ProjectType, required: false
+    argument :housing_type, Types::HmisSchema::Enums::HousingType, required: false
+    argument :tracking_method, Types::HmisSchema::Enums::TrackingMethod, required: false
+    argument :target_population, HmisSchema::Enums::TargetPopulation, required: false
+    argument :HOPWAMedAssistedLivingFac, HmisSchema::Enums::HOPWAMedAssistedLivingFac, required: false
+    yes_no_missing_argument :continuum_project, required: false
+    yes_no_missing_argument :residential_affiliation, required: false
+    yes_no_missing_argument :HMISParticipatingProject, required: false
+
+    def to_params
+      result = trim_keys(to_h.except(:organization_id))
```

Comment:
Why do we have to trim leading underscores (I think thats what its doing?)? I don't see why those would be present in the object keys

Reply:
Removed